src/cryptography/utils.py

```diff
@@ -46,6 +47,18 @@ def int_from_bytes(data, byteorder, signed=False):
         return result
 
 
+if hasattr(int, "to_bytes"):
+    int_to_bytes = int.to_bytes
+else:
+    def int_to_bytes(integer, byteorder, signed=False):
```

Comment:
Eh, fuck, `int.to_bytes` has an API where you need to include the desired length.